test/WebJobs.Script.Tests/Utils/StringUtilsTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.Utils;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Utils
+{
+    public sealed class StringUtilsTests
+    {
+        [Theory]
+        [InlineData("FeatureA,FeatureB,FeatureC", "FeatureB", ',', true)]
```

Comment:
Should we account for whitespace and empty tokens? 

"FeatureA, FeatureB,FeatureC"

Reply:
I intentionally left that out to maintain parity with the previous version, which also didn’t handle trimming. In my benchmark repository, I’ve included a unit test that compares the outputs of both implementations to ensure the functionality remains consistent.

https://github.com/kshyju/Benchmarks/blob/9fe638a43458591991317a72af4b7b7cd30ed6a9/tests/Benchmarks.Tests/Tests.cs#L17-L26